content/articles/gulp4-primer.md

```diff
@@ -0,0 +1,153 @@
+Title: Migrating to gulp 4 by example
+Date: 2015-03-13
+Short_summary: showing how to migrate to gulp 4 by example
+Category: Dev
+Authors: Vincent
+
+*Reading time: ~20 minutes.*
+
+Last year I wrote a blog post on my personal website (you can see it here [Gulp by example ](http://vincent.is/introducing-people-to-gulp/)) showing what [gulp](http://gulpjs.com/) was and making a demo project using it.
+Since then, the team behind it did a lot of work on v4 of gulp which, while not released as of the writing of the post, is stable enough to be used.  
+
+Rather than going through a list of new features, let's update my [angular boilerplate](https://github.com/Keats/ng-boilerplate) to use gulp4.
+
+You can have a look at the [Goal section of the README](https://github.com/Keats/ng-boilerplate#goal) to know what this boilerplate does but for those not wanting to open the link: Typescript, Sass, automatic DI, template preloading, testing with Karma and Protractor, live reload with browser-sync.
+> Note: I'm not using this boilerplate personally anymore since I don't really do vanilla angular anymore.
+
+Here's the [starting point](https://github.com/Keats/ng-boilerplate/tree/87e75551651e94dfc1aa6135e1ea7cb5bd61cf0f).
```

Comment:
I didn't understand what this link was about when it I read the post rendered as HTML.